packages/bundle/src/createCognitiveServicesSpeechServicesPonyfillFactory.js

```diff
@@ -69,7 +69,7 @@ export default function createCognitiveServicesSpeechServicesPonyfillFactory({
     //             We will not need this code when using Speech SDK 1.14.0 or up.
     // TODO: [P1] #3575 Remove the following lines when bumping to Speech SDK 1.14.0 or higher
     source.createAudioContext = () => {
-      if (!!source.privContext) {
+      if (!source.privContext) {
```

Comment:
This is to avoid linting error `no extra bool cast`

Reply:
But this also reverted the logic, right?

Reply:
You're right. I'll revert. Thanks.